tox.ini

```diff
@@ -43,6 +43,21 @@ deps =
 commands =
     bash -c "{toxinidir}/run-bindep.sh"
 
+[testenv:bandit]
+deps = -rtest-requirements.txt
+commands =
+# Ignore B404 about importing subprocess. We're knowingly and intentionally
+# using subprocess and don't need a warning about mere usage alone.
+# Specific warnings will still come through as they're found.
+#
+# Until bandit has the ability to log issues at lower severities than it's
+# configured to pass/fail based on, we need to run bandit twice. Run it
+# once with no severity or confidence level set to get everything, but ignore
+# that exit status. Then run it again on the highest severity and confidence
+# levels and get our pass/fail status from that.
+# This is reported to bandit at https://github.com/PyCQA/bandit/issues/341
+    - bandit -s B404 -i -l -r playbooks/ scripts/ gating/
```

Comment:
FYI: these runs complete in a matter of seconds, which is why duplicating them is currently the path forward. Depending on what happens on the issue submitted to bandit, if we can get it to one that's fine, but an alternate way to get what we want would mean writing some sort of bandit runner that runs it at the lowest level to gather everything to be logged, but then does some filtering/counting on its own to determine pass/fail status based on the levels.